### PR TITLE
Create PrestoJavaAggergationFuzzer to use Presto as source of truth.

### PR DESCRIPTION
**Pull request by kgpai**
This allows us to run aggregation fuzzer against Presto java. We are keeping it separate from AggregationFuzzer in order to : 
1. Have separate skip lists
2. Ensure we keep getting signal from aggregates against DDB while we try and get PrestoJava fuzzer stable.

### CODE REVIEW

File: velox/exec/tests/PrestoJavaAggregationFuzzerTest.cpp [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/exec/fuzzer/PrestoQueryRunner.h"
+#include "velox/exec/tests/AggregationFuzzerCommon.h"
+
+DEFINE_string(
+    presto_url,
```
Comment (duanmeng) [inline velox/exec/tests/PrestoJavaAggregationFuzzerTest.cpp:21]: Should we also add `gflags` to be able to config `user` and `timeout`?

File: velox/exec/tests/CMakeLists.txt [hunk 2]
```diff
@@ -200,18 +200,30 @@ target_link_libraries(
...
   gtest
   gtest_main)
 
+# Aggregation Fuzzer.
+add_executable(velox_aggregation_fuzzer_test AggregationFuzzerTest.cpp)
+
+target_link_libraries(velox_aggregation_fuzzer_test
+                      velox_aggregation_fuzzer_common)
+
+# Aggregation Fuzzer.
```
Comment (duanmeng) [inline velox/exec/tests/CMakeLists.txt:220]: s/Aggregation Fuzzer/PrestoJava Aggregation Fuzzer

**Review by duanmeng — COMMENTED** [review]
Looks great % some comments.

**Review by mbasmanova — CHANGES_REQUESTED** [review]
@kgpai It is confusing to have both xxxBase and xxxCommon. If the only change we need is to have a different skip list, then let's add GFlag to override skip list.

**Pull request closed without merging**